util/changelog.py

```diff
@@ -138,15 +138,21 @@ def __init__(self, args):
             if commit.sha == self.startCommit.sha:
                 break
             else:
-                for pull in commit.get_pulls():
+                message = commit.commit.message.partition('\n')[0]
+                try:
+                    pr_number = int(message[message.rfind('#')+1:message.rfind(')')])
+                    pull = self.repo.get_pull(pr_number)
                     labels = []
                     for label in pull.labels:
                         labels.append(label.name)
                     self.commits[pull.number] = {
                         "Title": pull.title,
                         "Url": pull.html_url,
                         "labels": labels
-                    }
+                        }
+                except ValueError:
+                    print("Commit has no associated PR {}: \"{}\"".format(commit.sha, message))
```

Comment:
Do we want to have this act as a catch for non-PRs, would still include the title and url, but the label would be other. An example of this would be when we increment the CMake versions strings, historically those have not been a specific PR but rather a push to the appropriate releases or develop branch?
Otherwise LGTM

Reply:
Great idea, pushing a commit that results in the following layout, what do you think?

### Other Updates
  
|Commit|Title  
|:-:|:--  
|[8263eb68](https://github.com/nanocurrency/nano-node/commit/8263eb681c70117b15e826a2e96a50cb261fa27e)|Changelog generator grab pull from commit message

Reply:
Handling with "other" label is quite difficult in the current script unfortunately.